include/Util/helper.php

```diff
@@ -72,4 +72,32 @@ function get_log_columns_to_export() {
 	}
 
 	return array( 'id', 'sent_date', 'to_email', 'subject' );
+}
+
+/**
+ * Returns TRUE if the User is Administrator or the User's role is allowed in Plugin's settings page.
+ *
+ * @since 2.1.0
+ *
+ * @return bool
+ */
+function can_current_user_email_log() {
+	$return_value = false;
+	$option       = get_option( 'el_email_log' );
+
+	if ( current_user_can( 'administrator' ) ) {
```

Comment:
Let's use guard conditions to make this method more readable.

You can read more about guard conditions at https://carlalexander.ca/mastering-php-conditionals/